Tools/autotest/sim_vehicle.py

```diff
@@ -549,7 +586,7 @@ def start_mavproxy(opts, stuff):
         extra_cmd += " " + stuff["extra_mavlink_cmds"]
 
     if opts.mavproxy_args:
-        cmd.extend(opts.mavproxy_args.split(" "))  # this could be a lot better..
+        cmd.extend(opts.mavproxy_args.split(" "))  # this could be a lot better
```

Comment:
Humm, so this comment doesn't need to be above the line like others do?

Reply:
Yep.